challenges/stats_preview_card_component/index.html

```diff
@@ -27,10 +27,70 @@
     />
     <link rel="manifest" href="../site.webmanifest" />
     <link rel="stylesheet" href="style.css" />
+    <link href="./dist/output.css" rel="stylesheet" />
   </head>
-  <body class="m-0">
-    <div
-      class="flex h-[100vh] w-full flex-col place-content-center text-center"
+
+  <body class="lg:bg-main-bg-color mx-auto bg-black sm:w-full lg:w-full">
```

Comment:
Same as in the other PR, no need for `sm:` prefix when doing mobile-first responsive design.